dynatrace/environment_v2/networkzones.py

```diff
@@ -0,0 +1,106 @@
+"""
+Copyright 2021 Dynatrace LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from datetime import datetime
+from dynatrace.dynatrace_object import DynatraceObject
+from typing import List, Optional, Union, Dict, Any
+
+from dynatrace.environment_v2.schemas import EntityType, ManagementZone
+from dynatrace.http_client import HttpClient
+from dynatrace.pagination import PaginatedList
+
+
+
+class NetworkZoneService:
+    ENDPOINT = "/api/v2/networkZones"
+    ENDPOINT_GLOBALSETTINGS = "/api/v2/networkZoneSettings"
+
+
+    def __init__(self, http_client: HttpClient):
+        self.__http_client = http_client
+
+
+    def list(self) -> PaginatedList["NetworkZone"]:
+        """ Lists all network zones. No params
+
+        :return: a list of Network Zones with details 
+        """
+        return PaginatedList(NetworkZone, self.__http_client, target_url=self.ENDPOINT, list_item="networkZones")
+
+
+    def get(self, networkzone_id: str):
+        """ Gets parameters of specified network zone
+
+        :param networkzone_id: the ID of the network zone
+        :return: a Network Zone + details
+        """
+        response = self.__http_client.make_request(f"{self.ENDPOINT}{networkzone_id}").json()
```

Comment:
I think you need to add a forward slash here `response = self.__http_client.make_request(f"{self.ENDPOINT}/{networkzone_id}").json()`

Reply:
This causes `Exception: Error making request to https://localhost:9999/e/eaa50379/api/v2/networkZonesdefault: <Response [404]>. Response: {"error":{"code":404,"message":"HTTP 404 Not Found"}}`